server/blob/blob-s3/pom.xml

```diff
@@ -82,6 +82,10 @@
             <groupId>io.projectreactor.addons</groupId>
             <artifactId>reactor-extra</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.projectreactor.addons</groupId>
+            <artifactId>reactor-pool</artifactId>
```

Comment:
This looks cool!

Maybe we want to evaluate the usage of this pool along with the rabbitMQ driver....

Reply:
sadly we cannot since the `Channel` leaks, which is not the case here

Reply:
I'm always impressed at how bad the java drivers for RabbitMQ are...